contracts/src/PDPService.sol

```diff
@@ -174,12 +174,52 @@ contract PDPService {
         return rootId;
     }
 
-    // Removes a root from a proof set. Must be called by the contract owner.
+    // removeRoot removes a root from a proof set. Must be called by the contract owner.
     function removeRoot(uint256 setId, uint256 rootId) public {
         require(proofSetOwner[setId] == msg.sender, "Only the owner can remove roots");
         // TODO: implement me
     }
 
+    // findRoot returns the root id for a given chunk index
```

Comment:
We'll need to know the leaf/chunk offset within the tree committed by the root as well. I expect that this can be found easily enough as part of this method.

Reply:
Great point I'll add this into the return now